books/free-programming-books-de.md

```diff
@@ -249,6 +248,5 @@
 
 ### Visual Basic
 
-* [Einstieg in Visual Basic 2010](http://openbook.rheinwerk-verlag.de/einstieg_vb_2010) - Thomas Theis (Online)
```

Comment:
You can replace it with: https://web.archive.org/web/20210507062754/https://openbook.rheinwerk-verlag.de/einstieg_vb_2010/

Reply:
How is it different from the 2012 version?

Reply:
2012 version has more chapters, all right